src/store/modules/product/getters.ts

```diff
@@ -7,5 +7,13 @@ const getters: GetterTree<ProductState, RootState> = {
     // Returning empty object so that it doesn't breaks the UI
     return state.cached[productId] ? state.cached[productId] : {};
   },
+  getSearchProducts: (state) => {
+    return state.products ? state.products : {};
```

Comment:
I think this is not needed. We are initializing with an empty array and re-initializing with empty. Is there any case it will have issues?

Reply:
no sir we can remove this empty array from here

Reply:
Removed